R/report.data.frame.R

```diff
@@ -475,10 +480,14 @@ report.numeric <- function(model, median = FALSE, centrality = TRUE, dispersion
   }
 
   # Missing
-  if (missing_percentage == TRUE) {
-    table <- dplyr::select(table, -one_of("n_Missing"))
-  } else {
-    table <- dplyr::select(table, -one_of("percentage_Missing"))
+  if(!is.null(missing_percentage)){
+    if (missing_percentage == TRUE) {
+      table <- dplyr::select(table, -one_of("n_Missing"))
```

Comment:
We have the small helper function `.remove_columns()` to avoid dplyr, already used elsewhere in _report_.